src/main/openapi/openapi.yaml

```diff
@@ -234,6 +237,12 @@ paths:
               schema:
                 type: string
                 format: binary
+        '500':
+          description: Parameters are probably out of range.
```

Comment:
Thats a `400: Bad Request`. The 400-499 range is for errors caused by the client. 500-599 are internal server errors.

Also: The automatic checking by OpenAPI is really bad as it does not tell the user much about what went wrong. Consider putting the check at the beginning of `LabapiService.exportImages` where the `ServiceExceptions` can be fully controlled.

Reply:
Addendum: I can see that you have done so in `ImageExport`, but I am afraid that it is too late in the chain? Try calling it with a too high max value and see what it says.